src/utils/util.js

```diff
@@ -0,0 +1,8 @@
+export default function isOnlyLeafs(children) {
```

Comment:
Not the best name, but I think we should have some general util file

Reply:
hasGrandChildren ?

Reply:
or in this way hasNoGrandChildren I guess

Reply:
onlyLeafChildren maybe? or hasOnlyLeafChildren?

Reply:
Ah... I was referring to the file name 😅The function name is pretty accurate imo. We are passing the children array and the function returns true if they all are leafs.

Reply:
Ah okay. Maybe tree-utils? In that case we could add this function also to the tree-transform util file and rename that one

Reply:
Sounds good, I'll do that